Assets/Scripts/UI/UITextOverlay.cs

```diff
@@ -48,6 +48,8 @@ public Coroutine SetText(string text)
 
     private IEnumerator StartTypeText(string text)
     {
+        if (GameManager.Instance.skipSettings.allTextCutscenes) yield break;
```

Comment:
Wouldn't this skip all convos since dialogue uses this script now?

Reply:
Ah, very good catch, that's true! I'll think of a better way to handle this.